src/assistants/openai/openai_assistant.rs

```diff
@@ -48,6 +49,7 @@ impl OpenAIAssistant {
             id: None,
             thread_id: None,
             run_id: None,
+            temperature: model.get_default_temperature(),
```

Comment:
Turns out we weren't even passing temperature when creating an assistant and where therefore always using the default (which is set to 1). With the changes in this file we now actually set the temperature and give users the ability to define it as they see fit.